UI/css/style.css

```diff
@@ -432,6 +432,10 @@ button:active {
     display: inline-block;
 }
 
+.navigation__listitem:not(:last-child) {
```

Comment:
Is it possible to update the css names to this format?
```
.navigation-list-item
```
For all selectors?

Reply:
That would mean changing it on all pages too
would do that  
thanks